### ISSUE TITLE
Closing Tabs takes me to another group

### ISSUE THREAD

**Issue by aashish108** [projectdelphai/panorama-tab-groups#193]
Hi! I have a rather annoying issue where, when closing tabs, sometimes I will get taken to another group for some reason. I then have to switch back. Does anyone know why this occurs pls?? It happens on my windows & mac Firefox v71.

I'm using panorama tab groups version 0.8.11. I have 107 tabs where only 7 are active.

Cheers

**Comment by roeyk** [comment]
Agreed.  This is probably the #1 most annoying bug that I have to deal with all the time.
